src/main/java/pl/nogacz/checkers/application/Design.java

```diff
@@ -64,6 +69,56 @@ public static void addLightMove(Coordinates coordinates) {
     }
 
     public static void removePawn(Coordinates coordinates) {
-        gridPane.getChildren().removeIf(node -> GridPane.getColumnIndex(node) == coordinates.getX() && GridPane.getRowIndex(node) == coordinates.getY());
+       gridPane.getChildren().removeIf(node -> GridPane.getColumnIndex(node) == coordinates.getX() && GridPane.getRowIndex(node) == coordinates.getY());
+    }
+
+    public static void changeScore(HashMap<Coordinates, PawnClass> board){

```

Comment:
Issue'da şöyle diyor

`the points for both sides can be calculated using a method in BoardPoint class.`

Bu PR bu ifadeye uyuyor mu?

Reply:
Hocam merhaba, ben kodda BoardPoint isminde bir class göremedim burada denmek istenen yeni bir class oluşturmam mı yoksa şuan bulunan Board class'ı mı ifade edilmek isteniyor acaba ? Yapacağım değişiklikleri buna göre yapacağım.

Reply:
Satranç projesinde vardı böyle bir class. Bu projede gerçekten yok. 

Aşağıdaki hesap çok sığ bir hesap değil mi? Sadece parçaları sayıyor ama parçaların pozisyonları ile ilgili herhangi bir hesap yapmıyor.

Bir checker board 'u verildiğinde kimin kazanmaya daha yakın olduğunu hesap eden daha sofistike bir algoritma kullanabilir misin? Eğer bir dış kaynaktan bulduysan kod içerisinde kaynağı belirtmeyi unutma.